src/pages/500.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import Error from 'next/error'
+
+export default function Custom500() {
+  return <Error statusCode={500} />
```

Comment:
@kmjennison saw this ticket was a 1 so I copied the return from the 404 page.  Is this what you were thinking for the 500 page?

Reply:
Yep! Just want a branded, less jarring error view.